app/views/certificates/i_belong/_achievement_list.html.erb

```diff
@@ -1,4 +1,4 @@
-<p class="govuk-body ncce-activity-list__title govuk-!-margin-top-8 <%= add_groups_complete_icon_class(current_user, @professional_development_groups) %>">
+<p id="<%= @professional_development_groups.first&.id %>" class="govuk-body ncce-activity-list__title govuk-!-margin-top-8 <%= add_groups_complete_icon_class(current_user, @professional_development_groups) %>">
```

Comment:
In production and dev-seeded environments, IB, Primary, and Secondary all have just one professional development group so its safe to take just the first one's ID.

We are adding these IDs in so we can hyperlink to different sections of the page in the Progress Bar

The reason I'm using `&.` is that some of our tests don't fully construct all of the programme activity groupings. This is generally just a greater issue with how the supporting data in our tests is put together and not something that I think is justified in being solved here

Reply:
If there's only one, could the controller just set `@professional_development_group` and be responsible for the `first` logic instead of it being a bit tucked away in this view?

Reply:
Or could the association even be updated so there's only ever one? Little bit smelly to have a data model that allows multiple of something where only one is relevant and used by the code.